test/pact/command/exec_command_test.exs

```diff
@@ -0,0 +1,528 @@
+defmodule Kadena.Pact.ExecCommandTest do
+  @moduledoc """
+  `ExecCommand` struct definition tests.
+  """
+  use ExUnit.Case
+
+  alias Kadena.Cryptography.KeyPair, as: CKeyPair
```

Comment:
@EdwinGuayacan @Odraxs Please avoid these contractions for clarity, use declarative names instead.